public/app/core/components/PageNew/PageHeader.tsx

```diff
@@ -11,7 +11,7 @@ export interface Props {
 
 export function PageHeader({ navItem, subTitle }: Props) {
   const styles = useStyles2(getStyles);
-  const sub = subTitle ?? navItem.subTitle;
+  const sub = subTitle ?? navItem.description;
```

Comment:
I would like to ask the case where we are going to use `subTitle` ? is there a case that `navItem.description` will not be available?

Reply:
naw, i want to consolidate all our mixed uses of `subTitle`/`description` to just `description` and we can remove `subTitle`.